src/lightning_app/components/multi_node/pl.py

```diff
@@ -0,0 +1,94 @@
+import os
+from dataclasses import dataclass
+from typing import Any, Callable, Type
+
+from typing_extensions import Protocol, runtime_checkable
+
+from lightning_app.components.multi_node.base import MultiNode
+from lightning_app.components.multi_node.pytorch_spawn import _PyTorchSpawnRunExecutor
+from lightning_app.core.work import LightningWork
+from lightning_app.utilities.app_helpers import is_static_method
+from lightning_app.utilities.packaging.cloud_compute import CloudCompute
+from lightning_app.utilities.tracer import Tracer
+
+
+@runtime_checkable
+class _PyTorchLightningProtocol(Protocol):
```

Comment:
same concerns as with the `_LiteProtocol` above.

Reply:
Renamed.